Wallet integration docs (#95)
Web3-Pi/web3-reverse-proxy: README.md

```

**IMPORTANT:** Resulting changes are saved in local `.env` file for reuse.

## Wallet integration

Any client can connet RPC Reverse Proxy. 
A web client, scripting tools, backend servers etc. 
It is the same as with other Ethereum RPC providers: you need to use a user's access URL containing API key.

> The proxy support CORS what enables usage within web browsers.

Reviewer: ```suggestion
The proxy supports CORS, which enables usage within web browsers.
```

> Users may wish to integrate with wallets, e.g. Metamask.

Reviewer: ```suggestion
Users may wish to integrate the proxy with wallet applications, e.g. Metamask.
```

> Below is the example of proper configuration.

Reviewer: ```suggestion
Below is the example of a proper configuration.
```

It is convenient to create a duplicate of Mainnet network configuration.

> The only data that is specific is RPC URL that is access URL and points the proxy.

Reviewer: ```suggestion
The only specific piece of setup is the RPC URL, which needs to be set to the proxy's RPC endpoint.
```

See the example below.

![Metamask](./admin/docs/metamask-1.png)

> In case of problems check, the best place to start the investigation is to check the network traffic,

Reviewer: ```suggestion
In case of any problems with the verification of the newly added network, the best place to start the investigation is to check the network traffic,
```

> for instance with a web browser's dev tools/developers tools (Ctrl+Shift+C).

Reviewer: ```suggestion
for instance with the web browser's dev tools/developers tools (Ctrl+Shift+C).
```
